kalite/inline/api_urls.py

```diff
@@ -0,0 +1,5 @@
+from django.conf.urls import include, patterns, url
+
+urlpatterns = patterns (__package__ + '.api_views',   
+    url(r'^(?P<narrative_id>\w+)', 'narrative_view', name="narrative-view")
```

Comment:
Check out this neat site: http://pythex.org/

In particular, `^(?P<narrative_id>[\w+/])` should match relative urls, since it includes the `/` character.